src/App.tsx

```diff
@@ -5,6 +5,40 @@ import { customersGetter, customersSetter } from './api/CustomerApi';
 import { initialCustomers } from './api/startingValues';
 import ContainerWrapper from './components/ContainerWrapper';
 import Landing from './pages/Landing';
+import { QueryClient, QueryClientProvider, useQueryClient, useMutation } from 'react-query';
+import App from './App';
+import ReactDOM from 'react-dom';
+
+const queryClient = new QueryClient();
+
+ReactDOM.render(
+  <QueryClientProvider client={queryClient}>
+    <App />
+  </QueryClientProvider>,
+  document.getElementById('root')
+);
```

Comment:
This shouldn't be needed, it's already implemented [here](https://github.com/content-services/React-typescript-demo/blob/main/src/index.tsx#L16C5-L16C47).